crates/sui-graphql-rpc/schema.graphql

```diff
@@ -448,6 +448,10 @@ type Checkpoint {
 	By default, the scanning range consists of all transactions in this checkpoint.
 	"""
 	transactionBlocks(first: Int, after: String, last: Int, before: String, filter: TransactionBlockFilter, scanLimit: Int): TransactionBlockConnection!
+	"""
+	The Base64 serialized BCS bytes of CheckpointSummary for this checkpoint.
+	"""
+	checkpointSummaryBcs: Base64
```

Comment:
I would just call this `bcs` (only need to give it a more descriptive name if there is something else this could feasibly be the BCS of).